x/stakeibc/keeper/msg_server_submit_tx.go

```diff
@@ -526,11 +526,6 @@ func (k Keeper) SubmitCalibrationICQ(ctx sdk.Context, hostZone types.HostZone, v
 		return errorsmod.Wrapf(err, "unable to marshal delegator shares callback data")
 	}
```

Comment:
it wont let me highlight it, but we can also remove the callback data from this

Reply:
another option is to share the same callback as the delegator_shares query and add a "calibration" bool field to the callback data to identify which one we're in. Then we can cut back on all the duplicated code

Reply:
Would you recommend the bool field? Seems cleaner, but I would prefer to not futz with the delegator_shares query as it's somewhat sensitive logic.

Reply:
yeah I think my vote is also to not touch it cause it's a pretty gnarly callback as is. I would just remove the callback data from the calibration query and call it a day. We can always try and clean it up later!

Reply:
@sampocs gentle bump

Reply:
Sorry not sure how I missed this earlier! You can just set it to `[]byte{}`!

Reply:
Thanks! Done [here](https://github.com/Stride-Labs/stride/pull/950/commits/299631329f48e246358fc2dd45d07c4591ff65ed).